docs/readmes/lte/GTPU_Extension_Header.md

```diff
@@ -0,0 +1,139 @@
+     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+     | Ext-Hdr Length|                                               |
+     +-+-+-+-+-+-+-+-+                                               |
+     |                  Extension Header Content                     |
+     .                                                               .
+     .                                               +-+-+-+-+-+-+-+-+
+     |                                               |  Next-Ext-Hdr |
+     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+In 5G SA, each flow is forwarded based on the appropriate QoS rules. QoS rules are configured by SMF as QoS profiles to UP components and these components perform QoS controls to PDUs based on rules. In downlink, a pipelineD pushes QFI into an extension header, and transmits the PDU to RAN.  In uplink, each UE obtains the QoS rule from SMF, and transmits PDUs with QFI containing the QoS rules to the RAN.
+
+### Transfer of PDU Session Information for Uplink Data Traffic
+
+The Transfer of PDU Session Information for uplink data packets involves transfer of control information elements related to the PDU Session from NG-RAN to Pipelined.
+
+The UL PDU SESSION INFORMATION frame includes a QoS Flow Identifier (QFI) field associated with the transferred packet.
+
+The below Information Elements present in the PDU Session Information frame:
+
+PDU Type: The PDU Type indicates the structure of the PDU session UP frame. The field takes the value of the PDU Type it identifies: "0" for PDU Type 0. Value range: {0= DL PDU SESSION INFORMATION, 1=UL PDU SESSION INFORMATION, 2-15=reserved for future PDU type extensions}
+Spare: The spare field is set to "0" by the sender and should not be interpreted by the receiver.
+QoS Flow Identifier: When this IE is present, this parameter indicates the QoS Flow Identifier of the QoS flow to which the transferred packet belongs.
+Padding: The padding is included at the end of the frame.
+
+### Transfer of PDU Session Information for Downlink Data Traffic
+
+The Transfer of PDU Session Information for downlink data packets involves transfer of control information elements related to the PDU Session from Pipelined to NG-RAN.
+
+The DL PDU SESSION INFORMATION frame includes a QoS Flow Identifier (QFI) field associated with the transferred packet. The NG-RAN uses the received QFI to determine the QoS flow and QoS profile which are associated with the received packet.
+
+The DL PDU session information frame includes the Reflective QoS Indicator (RQI) field to indicate whether the user plane reflective QoS is to be activated or not. This is only applicable if reflective QoS is activated.
+
+## High Level Design
+
+The following functionality is supported for QFI set in GTP Extension header:
+QFI Configuration for uplink flow (Matching criteria)

```

Comment:
This should be made into a list by e.g. adding dashes to the start of each line, otherwise they're all just added together. Right now, it looks like "The following functionality is supported for QFI set in GTP Extension header: QFI Configuration for uplink flow (Matching criteria) QFI Configuration for downlink flow (Action criteria) Uplink traffic match with uplink flow Downlink traffic takes action with action criteria values."

Reply:
Fixed it.